app/resources/1.1/material_artwork/sparql/premis_ie.sparql

```diff
@@ -54,35 +83,288 @@ WHERE {
                 }
             }
             BIND(iri(concat(str(haObjId:), ?local_id_type)) AS ?local_id_type_uri)
-            BIND(iri(concat(str(haObjId:), concat(?local_id_type, ENCODE_FOR_URI(?local_id_val)))) AS ?local_id)
+            BIND(iri(concat(str(haObjId:), MD5(concat(?local_id_type, ?local_id_val)))) AS ?local_id)
```

Comment:
Waarom overgeschakeld naar een MD5 hash?

Reply:
Er zaten dots in de value die niet goed werkten bij een shacl validatie.